packages/angular-workspace/nimble-angular/src/directives/checkbox/nimble-checkbox-control-value-accessor.directive.ts

```diff
@@ -10,14 +11,30 @@ import { CheckboxControlValueAccessor } from '../../thirdparty/directives/checkb
 @Directive({
     selector:
       'nimble-checkbox[formControlName],nimble-checkbox[formControl],nimble-checkbox[ngModel]',
-    // The following host metadata is duplicated from CheckboxControlValueAccessor
     // eslint-disable-next-line @angular-eslint/no-host-metadata-property, @typescript-eslint/naming-convention
-    host: { '(change)': 'onChange($event.target.checked)', '(blur)': 'onTouched()' },
+    host: { '(blur)': 'onTouched()' },
     providers: [{
         provide: NG_VALUE_ACCESSOR,
         useExisting: forwardRef(() => NimbleCheckboxControlValueAccessorDirective),
         multi: true
     }]
 })
 export class NimbleCheckboxControlValueAccessorDirective extends CheckboxControlValueAccessor {
+    private ignoreChangeEvents = false;
```

Comment:
Can we create / link to an existing bug around checkbox change events firing on programmatic changes?
Sounds very familiar so may already exist. Can reference that issue here to remove the workaround when fixed.